### DISCUSSION TITLE
Have Changed-files as a verified GH action

### DISCUSSION THREAD

**Discussion by ritaCanavarro** [tj-actions/changed-files#1928] [Q&A] [answered]
Hello team,

We are currently using this action, but due to some internal policy requirements to keep using it we need to have it verified.
Is it possible to have this action as being verified by GH?

**Comment by jackton1** [comment, marked as answer]
Hello @ritaCanavarro,

Thank you for your question. Currently, GitHub has not opened up a general verification process for GitHub Actions. Currently, the verification process is unavailable for actions created by the community. 

For more detailed information, you can refer to this discussion, which provides insights into the current issue:
- https://github.com/orgs/community/discussions/25265

If you have any further questions or need additional assistance, feel free to ask.

**Reply by ritaCanavarro** [reply]
Hi @jackton1 ,

Thanks for the answer, it is unfortunate that there is still not a process for the community.
